walkthrough/garden_cookiecutter/{{cookiecutter.directory_name}}/{{cookiecutter.short_name}}.py

```diff
@@ -11,6 +11,26 @@
 
 # Get paths and naming conventions for current step.
 paths = PathFinder(__file__)
+{% if cookiecutter.load_countries_regions == "True" %}
+
+
+def load_countries_regions() -> Table:
+    """Load countries-regions table from reference dataset (e.g. to map from iso codes to country names)."""
+    ds_reference: Dataset = paths.load_dependency("reference")
+    tb_countries_regions = ds_reference["countries_regions"]
+
+    return tb_countries_regions
+{% endif -%}
+{% if cookiecutter.load_population == "True" %}
+
+
+def load_population() -> Table:
```

Comment:
This function refers to the old `key_indicators` dataset, so when using it, it will break

Reply:
oh wow, almost missed that. Thanks!